gst/droidcamsrc/gstdroidcamsrcdev.c

```diff
@@ -461,6 +461,8 @@ gst_droidcamsrc_dev_frame_available (void *user, DroidMediaBuffer * buffer)
 
   droid_media_buffer_get_info (buffer, &info);
 
+  rect = droid_media_buffer_get_crop_rect (buffer);
```

Comment:
Does `droid_media_buffer_get_crop_rect ()` internally acquire any locks? If not then it could be just
```c
  GST_OBJECT_LOCK (src);
  src->crop_rect = droid_media_buffer_get_crop_rect (buffer);
  GST_OBJECT_UNLOCK (src);
```
without a temporary variable.

Reply:
The rect variable is used later in the function in the gst_droidcamsrc_dev_prepare_buffer call.